MicroAOD/python/flashggDiPhotons_cfi.py

```diff
@@ -12,8 +12,8 @@
                                   GenParticleTag         = cms.InputTag( "flashggPrunedGenParticles" ),
 
                                   ##Parameters for Legacy Vertex Selector                                                
-                                  vertexIdMVAweightfile   = cms.FileInPath(""),
-                                  vertexProbMVAweightfile = cms.FileInPath(""),
+                                  vertexIdMVAweightfile   = cms.FileInPath("flashgg/MicroAOD/data/TMVAClassification_BDTVtxId_SL_2016.xml"),
```

Comment:
You should get the weight file name from meta conditions, where it is already available.

Reply:
Ok, makes sense. I will change this.

Reply:
thanks, I will also make a script to make the weight file "persistent" by copying them from EOS in case EOS availability is an issue.